CHANGELOG.md

```diff
@@ -78,6 +78,7 @@
 * [#4521](https://github.com/bbatsov/rubocop/issues/4521): Move naming related cops into their own `Naming` department. ([@drenmi][])
 * [#4600](https://github.com/bbatsov/rubocop/pull/4600): Make `Style/RedundantSelf` aware of arguments of a block. ([@Envek][])
 * [#4658](https://github.com/bbatsov/rubocop/issues/4658): Disable auto-correction for `Performance/TimesMap` by default. ([@Envek][])
+* Remove unnecessary pry imports. ([@b-t-g][])
```

Comment:
That's not a user visible change, so there should not be a changelog entry.

Reply:
Makes sense, I was on the fence whether to add a changelog entry, but opted to just add it. Removed!